views/play.erb

```diff
@@ -0,0 +1,30 @@
+<style>
+
+  .content{
+    position: fixed;
+  top: 50%;
+  left: 50%;
```

Comment:
Maybe you could move all CSS into an external CSS file you can load with `<link ... />` to have it separated from the HTML